homeassistant/components/switch/tplink.py

```diff
@@ -0,0 +1,55 @@
+"""Support for TPLink HS100/HS110 smart switch.
+
+It is able to monitor current switch status, as well as turn on and off the switch. 
+
+"""
+
+import logging
+
+from homeassistant.components.switch import SwitchDevice
+from homeassistant.const import (
+    CONF_HOST, CONF_NAME)
+
+# constants
+DEVICE_DEFAULT_NAME = 'HS100'
+REQUIREMENTS = ['http://github.com/gadgetreactor/pyHS100/archive/'
+                '27dd078e57628e76a1de89cc9d81b703f8e846d1'
```

Comment:
This doesn't work. This archive cannot be installed. You need to add the `.zip` extension and your package needs to contain a `setup.py` that describes your package. [See this example](https://github.com/Xorso/pyalarmdotcom/blob/master/setup.py)